Gruntfile.js

```diff
@@ -181,7 +181,8 @@ module.exports = function(grunt) {
     */
   grunt.registerTask('nose', ['shell:nose-chrome', 'shell:nose-ie10']);
   grunt.registerTask('test', ['eslint', 'mocha_istanbul', 'nose']);
-  grunt.registerTask('build', ['squish', 'test']);
+  grunt.registerTask('test-js', ['eslint', 'mocha_istanbul']);
```

Comment:
'test-js' task could call 'test' to make it [DRY](http://c2.com/cgi/wiki?DontRepeatYourself)-er